ramls/invoice.raml

```diff
@@ -176,6 +176,21 @@ resourceTypes:
         description: Update an invoice line with given {id}
       delete:
         description: Delete an invoice line with given {id}
+    /fundDistributions/validate:
```

Comment:
In proposed implementation endpoint path would be like this: /invoice-lines/fundDistributions/validate
So there is mixing of styles, we need to use consistent approach in naming, so you need to change path to this one: /invoice-lines/fund-distributions/validate
Also it is good to be consisted with mod-orders where the same naming convention chosen:
https://github.com/folio-org/mod-orders/blob/master/descriptors/ModuleDescriptor-template.json#L311

Reply:
yup, i agree.